test/performance_tests/performance_tests.vcxproj

```diff
@@ -1,183 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You removed project files for Visual Studio which causes compilation failure for AppVeyor. Please return them back and if it is possible we need to generate project files within CMake before compiling the project on AppVeyor.

Reply:
I updated the appveyor.yml to generate automatically Visual Studio solutions. Hope it will work.

Reply:
Yes, it works. We still need to revert back VS project files as not every person want to use CMake. We will remove them later after CUDA and OpenCL integration.